### PR TITLE
Bump version to 0.13.0

### PR DESCRIPTION
**Pull request by syall**
*Issue #, if available:*

N/A

*Description of changes:*

Bump version to 0.13.0.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by syall** [comment]
Pending commit with `CHANGELOG.md` changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,69 @@
...
+* Fix Error printout for protocol-response tests ([#657](https://github.com/awslabs/smithy-typescript/pull/657))
+* Read staticContextParams from operation ([#658](https://github.com/awslabs/smithy-typescript/pull/658))
+* Fix codegen windows ([#661](https://github.com/awslabs/smithy-typescript/pull/661))
+* Fix missing underscore prefix in DefaultsMode Provider ([#671](https://github.com/awslabs/smithy-typescript/pull/671))
+
+### Miscellaneous
+
+* Update versions in README example ([#601](https://github.com/awslabs/smithy-typescript/pull/601))
+* Update links to point to smithy.io ([#637](https://github.com/awslabs/smithy-typescript/pull/637))
+* Add smithy as codeowners for the whole repo ([#664](https://github.com/awslabs/smithy-typescript/pull/664))
```
Comment (gosar) [inline CHANGELOG.md:63]: These are internal repo setup. They don't need to go into changelog. Only keep stuff that would be relevant to consumers of this package. Look through all the PRs in this change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,69 @@
 # Smithy Typescript Codegen Changelog
 
+## 0.13.0 (2023-01-25)
+
+### Features
+
+* feat(smithy-typescript-codegen): allow deferred resolution for api key config ([#588](https://github.com/awslabs/smithy-typescript/pull/588))
```
Comment (gosar) [inline CHANGELOG.md:7]: remove `feat`, `chore`, `fix` from here. The entries here may need to be edited from the PR titles.

**Review by gosar — COMMENTED** [review]
See if related PRs can be combined into a single line item that summarizes a set of related PRs.

**Review by AndrewFossAWS — APPROVED** [review]

**Review by trivikr — APPROVED** [review]

**Pull request merged**